packages/@storefront/flux-capacitor/test/unit/core/adapters/configuration.ts

```diff
@@ -280,6 +280,24 @@ suite('Configuration Adapter', ({ expect, stub }) => {
     });
   });
 
+  describe('extractToggleNavigations()', () => {
+    it('should return the toggle navigations', () => {
```

Comment:
Is there a 'type' of navigations that this function should ignore? If so, lets test for that.

Reply:
Since the update test case includes navigation types *other* than 'toggle', this is addressed.